src/components/plots/rectanglePlot.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../../reference.ts" />
+
+module Plottable {
+export module Plot {
+  export class Rectangle<X, Y> extends AbstractXYPlot<X, Y> {
+
+    /**
+     * Constructs a RectanglePlot.
+     *
+     * A RectanglePlot consists of a bunch of rectangles. The user is required to
+     * project the left and right bounds of the rectangle (x1 and x2 respectively)
+     * as well as the bottom and top bounds (y1 and y2 respectively)
+     *
+     * @constructor
+     * @param {Scale.AbstractScale} xScale The x scale to use.
+     * @param {Scale.AbstractScale} yScale The y scale to use.
+     */
+    constructor(xScale: Scale.AbstractScale<X, any>, yScale: Scale.AbstractScale<Y, any>) {
+      super(xScale, yScale);
+      this.classed("rectangle-plot", true);
+    }
+
+    protected _getDrawer(key: string) {
+      return new _Drawer.Rect(key, true);
+    }
+
+    protected _generateAttrToProjector() {
+      var attrToProjector = super._generateAttrToProjector();
+
+      // Copy each of the different projectors.
+      var x1Attr = attrToProjector["x1"];
+      var y1Attr = attrToProjector["y1"];
```

Comment:
Similar to how scatterPlot used to implement attrToProjector, we should probably pull the x/y attrToProjectors and then rename them to use x1/x2

https://github.com/palantir/plottable/blob/master/src/components/plots/scatterPlot.ts#L44